app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md

```diff
@@ -4,8 +4,8 @@ heading: If you want to change career
 description: |-
   Find out what extra support you can get if you want to change career and get into teaching. Bring your experience to life in the classroom.
 related_content:
-    From construction site to classroom : "/my-story-into-teaching/career-changers/from-construction-site-to-classroom"
-    Swapping senior management for students : "/my-story-into-teaching/career-changers/swapping-senior-management-for-students"
+    From engineering to teaching physics: "/blog/natural-transition-from-engineering-to-teaching-physics"
```

Comment:
the order of these links seems a little weird - two links re engineering to physics but separated?

one for @EllieNodder !

Reply:
I'd agree, and I'd also say the link text needs rewording - don't feel like they're differentiated enough

Reply:
Should we drop one of them?